### ISSUE TITLE
ReferenceError: input is not defined

### ISSUE THREAD

**Issue by freedevelopme** [olamigokayphils/blockio-multisig-sweep#1]
Hi. 
Getting the error: ReferenceError: input is not defined
What that mean?

**Comment by freedevelopme** [comment]
{ n: '10' }
ReferenceError: input is not defined

**Comment by olamigokayphils** [comment]
Hello @freedevelopme, this has been addressed: https://github.com/olamigokayphils/blockio-multisig-sweep/commit/90ef361de21292084a3600f3857fab1c73764232

Kindly let me know if any other issue persist.

**Comment by freedevelopme** [comment]
Next error: Error: Error: TypeError: Only absolute URLs are supported

**Comment by olamigokayphils** [comment]
Mind sharing the Log before this error? Do you have Transaction Hex?

**Comment by freedevelopme** [comment]
Error: Error: TypeError: Only absolute URLs are supported
    at ProviderService.sendTx (/Users/user/blockio-multisig-sweep/src/services/ProviderService.js:108:11)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async BlockIoSweep.begin (/Users/user/blockio-multisig-sweep/src/sweeper.js:194:7)
    at async Sweep (/Users/user/blockio-multisig-sweep/index.js:24:5)
Error: Error: Error: TypeError: Only absolute URLs are supported
    at BlockIoSweep.begin (/Users/user/blockio-multisig-sweep/src/sweeper.js:198:11)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Sweep (/Users/user/blockio-multisig-sweep/index.js:24:5)
    
    Do you have Transaction Hex? - Yes I do.

**Comment by olamigokayphils** [comment]
Awesome. While I fix this. Kindly leverage BlockCypher web portal. https://live.blockcypher.com/btc/pushtx/
 - Copy your Transaction Hex
 - Paste In & Broadcast to the Network.

 I'll share an update on the API notification for others once that is done.

**Comment by olamigokayphils** [comment]
If Broadcasted successfully Please comment here. So I can close this issue. Then open a new issue for **ProviderService.sendTx Error**

**Comment by freedevelopme** [comment]
Thank you so much!

**Comment by freedevelopme** [comment]
Yes it is
